components/Side/Bar.vue

```diff
@@ -1,4 +1,216 @@
+              fill="#333"
+              class="group-hover:fill-white"
+            ></path>
+            <path
+              d="M2 12C2 7.28595 2 4.92893 3.46447 3.46447C4.92893 2 7.28595 2 12 2C16.714 2 19.0711 2 20.5355 3.46447C22 4.92893 22 7.28595 22 12C22 16.714 22 19.0711 20.5355 20.5355C19.0711 22 16.714 22 12 22C7.28595 22 4.92893 22 3.46447 20.5355C2 19.0711 2 16.714 2 12Z"
+              stroke="#333"
+              stroke-width="1.5"
+              class="group-hover:stroke-white"
+            ></path>
+          </g>
+        </svg>
+        <p :class="$style.text">Dashboard</p>
```

Comment:
For dashboard you could use more explicit icon, ie: https://www.svgrepo.com/svg/520684/dashboard